regression/rt_ar_react_1x2v_quadraticV.c

```diff
@@ -0,0 +1,969 @@
+          .ion_mass = ctx.massAr, 
+          .elc_mass = ctx.massElc, 
+        }, 
+        { .react_id = GKYL_REACT_RECOMB,
+          .type_self = GKYL_SELF_RECVR,
+          .ion_id = GKYL_ION_AR,
+          .elc_nm = "elc",
+          .ion_nm = "Ar2",
+          .recvr_nm = "Ar1",
+          .charge_state = 1,
+          .ion_mass = ctx.massAr,
+          .elc_mass = ctx.massElc,
+        },
+      },
+    },
+    .bcx = {
+      .lower = { .type = GKYL_SPECIES_ZERO_FLUX, },
+      .upper = { .type = GKYL_SPECIES_ZERO_FLUX, },
+    },
+
+
+    /*    .bcx = {
+       .lower = {.type = GKYL_SPECIES_GK_SHEATH, },
+       .upper = {.type = GKYL_SPECIES_GK_SHEATH, },
+       },*/
+    
+    .num_diag_moments = 5,
+    .diag_moments = { "M0", "M1", "M2", "M2par", "M2perp" },
+  };
+
+  // Ar2+ ions
+  struct gkyl_gyrokinetic_species Ar2 = {
+    .name = "Ar2",
+    .charge = 2*ctx.chargeIon, .mass = ctx.massAr,
+    .lower = { -ctx.vpar_max_Ar, 0.0},
+    .upper = { ctx.vpar_max_Ar, ctx.mu_max_Ar}, 
+    .cells = { NV, NMU },
+    .polarization_density = ctx.n0Ar2,
+
+    .projection = {
+      .proj_id = GKYL_PROJ_MAXWELLIAN_PRIM, 
+      .ctx_density = &ctx,
+      .density = eval_density_ar2,
+      .ctx_upar = &ctx,
+      .upar= eval_upar,
+      .ctx_temp = &ctx,
+      .temp = eval_temp_ar,      
+    },
+
+    .collisions =  {
+      .collision_id = GKYL_LBO_COLLISIONS,
+      .normNu = true,
+      .n_ref = ctx.n0Ar2,
+      .T_ref = ctx.TAr,
+      .ctx = &ctx,
+      .self_nu = evalNuIon,
+      .num_cross_collisions = 3,
+      .collide_with = { "elc", "ion", "Ar1" },
+    },
+
+        .react = {
+      .num_react = 2,
+      .react_type = {
+        { .react_id = GKYL_REACT_IZ, 
+          .type_self = GKYL_SELF_ION, 
+          .ion_id = GKYL_ION_AR, 
+          .elc_nm = "elc", 
+          .ion_nm = "Ar2", 
+          .donor_nm = "Ar1", 
+          .charge_state = 1, 
+          .ion_mass = ctx.massAr, 
+          .elc_mass = ctx.massElc, 
+        }, 
+        { .react_id = GKYL_REACT_RECOMB,
+          .type_self = GKYL_SELF_ION,
+          .ion_id = GKYL_ION_AR,
+          .elc_nm = "elc",
+          .ion_nm = "Ar2",
+          .recvr_nm = "Ar1",
+          .charge_state = 1,
+          .ion_mass = ctx.massAr,
+          .elc_mass = ctx.massElc,
+        },
+      },
+      },
+    
+    .bcx = {
+      .lower = { .type = GKYL_SPECIES_ZERO_FLUX, },
+      .upper = { .type = GKYL_SPECIES_ZERO_FLUX, },
+    },
+
+    /*
+    .bcx = {
+       .lower = {.type = GKYL_SPECIES_GK_SHEATH, },
+       .upper = {.type = GKYL_SPECIES_GK_SHEATH, },
+    },
+    */
+    .num_diag_moments = 5,
+    .diag_moments = { "M0", "M1", "M2", "M2par", "M2perp" },
+  };
+  
+  // field
+  struct gkyl_gyrokinetic_field field = {
+    .fem_parbc = GKYL_FEM_PARPROJ_PERIODIC, 
+    .kperpSq = pow(ctx.kperp, 2.),
+  };
+
+  // GK app
+  struct gkyl_gk gk = {
+    .name = "ar_react_1x2v_quad_wJacobvel",
+
+    //.cfl_frac = 0.1,
+
+    .cdim = 1, .vdim = 2,
```

Comment:
@jRoeltgen see how cdim and vdim are specified in established codes, or specifically in this branch
https://github.com/ammarhakim/gkylzero/pull/469